File: INTRODUCCION-POO/EJERCICIO-1/Persona.java
class Persona {
    String nombre;
    int edad;
    String ciudad;

    public Persona(String nombre, int edad, String ciudad) {
        this.nombre = nombre;
        this.edad = edad;
        this.ciudad = ciudad;
    }

    public String saludo() {
        return "Hola, soy " + this.nombre + " de " + this.ciudad;
    }

    public String esMayorDeEdad() {
        if (this.edad >= 18) {
            return "Verdad";
        } else {
            return "Falso";
        }
    }

    public static void main(String[] args) {
        // b) Crea tres personas y muestra su saludo
        Persona persona1 = new Persona("Ana Maria", 30, "La Paz");
        Persona persona2 = new Persona("Cesar", 15, "Cochabamba");
        Persona persona3 = new Persona("Sonia", 25, "Santa Cruz");

        System.out.println(persona1.saludo());
        System.out.println(persona2.saludo());
        System.out.println(persona3.saludo());
        System.out.println("***************************************************");

        // c) Agrega un método para verificar si es mayor de edad
        System.out.println(persona1.nombre + " es mayor de edad: " + persona1.esMayorDeEdad());
        System.out.println(persona2.nombre + " es mayor de edad: " + persona2.esMayorDeEdad());
        System.out.println(persona3.nombre + " es mayor de edad: " + persona3.esMayorDeEdad());
    }
}

File: INTRODUCCION-POO/EJERCICIO-1/Persona.py
class Persona:
    def __init__(self, nombre, edad, ciudad):
        self.nombre = nombre
        self.edad = edad
        self.ciudad = ciudad

    def saludo(self):
        return f"Hola, soy {self.nombre} de {self.ciudad}"

    def es_mayor_de_edad(self):
        if self.edad >= 18:
            return "Verdad"
        else:
            return "Falso"

# b) Crea tres personas y muestra su saludo
persona1 = Persona("Ana Maria", 30, "La Paz")
persona2 = Persona("Cesar", 15, "Cochabamba")
persona3 = Persona("Sonia", 25, "Santa Cruz")

print(persona1.saludo())
print(persona2.saludo())
print(persona3.saludo())
print("***************************************************")
# c) Agrega un método para verificar si es mayor de edad
print(f"{persona1.nombre} es mayor de edad: {persona1.es_mayor_de_edad()}")
print(f"{persona2.nombre} es mayor de edad: {persona2.es_mayor_de_edad()}")
print(f"{persona3.nombre} es mayor de edad: {persona3.es_mayor_de_edad()}")
